denylist.go

```diff
@@ -774,66 +818,40 @@ func (dl *Denylist) isIPFSIPLDPathBlocked(cidStr, subpath, protocol string) Stat
 	}
 
 	prefix := c.Prefix()
-	for mhCode, blocks := range dl.DoubleHashBlocksDB {
-		// <cidv1base32>/<path>
-		// TODO: we should be able to disable this part with an Option
-		// or a hint for denylists not using it.
-		v1b32 := cid.NewCidV1(prefix.Codec, c.Hash()).String() // base32 string
-		v1b32path := v1b32
-		// badbits appends / on empty subpath. and hashes that
-		// https://github.com/protocol/badbits.dwebops.pub/blob/main/badbits-lambda/helpers.py#L17
-		v1b32path += "/" + subpath
-		doubleLegacy, err := multihash.Sum([]byte(v1b32path), mhCode, -1)
-		if err != nil {
-			// Usually this means an unsupported hash function was
-			// registered.
-			logger.Error(err)
-			continue
-		}
-
-		// encode as b58 which is the key we use for the BlocksDB.
-		b58 := doubleLegacy.B58String()
-		logger.Debugf("IsIPFFSIPLDPathBlocked load IPFS doublehash (legacy): %d %s", mhCode, b58)
-		entries, _ := blocks.Load(b58)
-		status, entry := entries.CheckPathStatus("")
-		if status != StatusNotFound { // Hit!
-			return StatusResponse{
-				Path:     p,
-				Status:   status,
-				Filename: dl.Filename,
-				Entry:    entry,
-			}
+	// Checks for legacy doublehash blocking
+	// <cidv1base32>/<path>
+	// TODO: we should be able to disable this part with an Option
+	// or a hint for denylists not using it.
+	v1b32 := cid.NewCidV1(prefix.Codec, c.Hash()).String() // base32 string
```

Comment:
nit: just in case someone, years from now, changes default base in go-cid ;)
```suggestion
	v1b32 := cid.NewCidV1(prefix.Codec, c.Hash()).StringOfBase(mbase.Base32) // base32 string
```